src/scenes/MerchStorePage/SingleItemPage.tsx

```diff
@@ -43,37 +47,40 @@ export default function ItemPage({
   const isMobile = useIsMobileOrMobileLargeWindowWidth();
 
   return (
-    <StyledPage pushItemsDown={isMobile && userOwnedSupply > 0}>
-      <StyledWrapper>
-        <StyledImageContainer>
-          <FlippingImage src={image} />
-        </StyledImageContainer>
-        <StyledContent>
-          <TitleM>
-            {title} {label}
-          </TitleM>
-          <BaseM>{description}</BaseM>
-          {!isMobile && <Spacer height={8} />}
-          <StyledPriceQuantityAndPurchaseContainer>
-            <StyledPriceAndQuantity>
-              <StyledPrice>{ethers.utils.formatEther(tokenPrice)} Ξ each</StyledPrice>
-              <BaseM>
-                {typeof publicSupply == 'number' && typeof usedPublicSupply == 'number'
-                  ? `${publicSupply - usedPublicSupply} / ${publicSupply} left`
-                  : ''}
-              </BaseM>
-            </StyledPriceAndQuantity>
-            {!isMobile && <Spacer height={16} />}
-            <PurchaseBox
-              label={label}
-              tokenId={tokenId}
-              // We don't want to disable the button if the user is not logged in. So we don't simply check equality between these two variables, which will be undefined if the user is not logged in.
-              disabled={publicSupply - usedPublicSupply == 0}
-            />
-          </StyledPriceQuantityAndPurchaseContainer>
-        </StyledContent>
-      </StyledWrapper>
-    </StyledPage>
+    <>
+      <StyledConnectedAddress>{address}</StyledConnectedAddress>
```

Comment:
just added this line.
rest of change is just wrapping everything in <></>.

Ideally would have put the connected address within the StyledPage but that would require modifying the layout of StyledPage to be full page width, height etc. Didn't want to cause a conflict or regressions at this stage given the timing and small scope of this change, so I just added the address to be position: absolute and rendered above the existing page content so that it can be positioned at the top corner.